pkgs/desktops/lxqt/pcmanfm-qt/default.nix

```diff
@@ -42,6 +42,13 @@ mkDerivation rec {
 
   passthru.updateScript = lxqtUpdateScript { inherit pname version src; };
 
+  preFixup = ''
+    sed -i 's|^\(Wallpaper=\)/nix/store/.*\(/lxqt-origami-light.png)$|\1/run/current-system/sw/share/lxqt/themes/frost\2|' $out/share/pcmanfm-qt/lxqt/settings.conf
```

Comment:
It is missing a `\` just before `)$`.

But it may be better to change `config/pcmanfm-qt/lxqt/settings.conf.in` in the source tree in the `postPatch` phase. Replace  `@LXQT_SHARE_DIR@` by `/run/current-system/sw/share/lxqt`.

Reply:
Yes, that seems a better place for patching.